src/gui/states.py

```diff
@@ -242,6 +242,18 @@ def gui_callback(self, gui_event):
 
             self.application.notbusy()
 
+        elif event == GuiEvent.PRINT_CHEMICAL_LABEL:
+
+            self.application.busy()
+
+            label = label_creator.create_chemical_storage_label(self.member.member_number, self.member.get_name())
```

Comment:
`label` is an image. Perhaps call it `label_image`?
```suggestion
            label_image = label_creator.create_chemical_storage_label(self.member.member_number, self.member.get_name())
```